src/server/master_server/lock_manager.cc

```diff
@@ -31,46 +31,58 @@ absl::Mutex* LockManager::AddLockIfNonExist(const std::string& pathname) {
   return filePathLocks_[idx].at(pathname);
```

Comment:
I prefer to use at when possible because [] can insert an item if the key does not exist. In this specific they are equivalent because the existence has been checked.